eng/DownloadSharedSource.ps1

```diff
@@ -18,7 +18,7 @@ $baseUrl = 'https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk
 DownloadAll $files $baseUrl $downloadPath
 
 #Download management Shared
-$files = 'ClientContext.cs'
+$files = 'ClientContext.cs', 'SharedExtensions.cs'
```

Comment:
Where are we using 'SharedExtensions.cs' internal class?

Reply:
it has the methods that are used by each RP but are now made internal since we didn't want to expose them as public APIs